### DISCUSSION TITLE
Release Date, compatability

### DISCUSSION THREAD

**Discussion by CheeseApprentice** [DaXcess/LCVR#69] [Q&A] [answered]
I know you answered that it will be on thunderstore but do you have a general Idea when? Also is computer compatable with VR or only VR and VR, Thank you! Im very excited to play this

**Comment by DaXcess** [comment, marked as answer]
Completely cross compatible, and 3 PM CEST

**Reply by CheeseApprentice** [reply]
I love you, i cant wait to get murdered in vr
